package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "homebridge",
   "description": "HomeKit support for the impatient",
-  "version": "0.4.50",
+  "version": "0.4.51",
   "scripts": {
     "dev": "DEBUG=* ./bin/homebridge -D -P example-plugins/ || true"
   },
   "author": {
-    "name": "Nick Farina"
+    "name": "Homebridge"
```

Comment:
I think it's better to keep @nfarina as an author, since it's metadata for npm and used for number of purposes,
and just decouple it from Manufacturer field in the Homebridge itself. Either hardcode it or read it from some config file.

Reply:
@alexindigo i will make it look at the config.